containers/multisite/wp-config.php

```diff
@@ -93,14 +94,13 @@
 }
 
 /* Multisite */
-
-// define('WP_ALLOW_MULTISITE', true);
-define('MULTISITE', true);
+define('WP_ALLOW_MULTISITE', true);
```

Comment:
@bmanth60 oh yeah, I didn't think about how my use of a `docker` host alias might affect the portability of this. Unfortunately, multisite installs are strongly tied to the domains used when creating the site.

Making this more portable is best saved for another issue, but your solution of adding `docker` to your hosts file should fix the issue for you.

Added https://github.com/ktbartholomew/saml-20-single-sign-on/issues/9